### DISCUSSION TITLE
Auto-join / shout more for room upgrades

### DISCUSSION THREAD

**Discussion by jryans** [element-hq/element-meta#832] [Ideas]
At the moment, when a room is upgraded, I get the default "noisy" notification on web. However, the room is not binged red, so it's quite hard to find among all the unread rooms. At the same time, I have to go find the room and manually click to follow to join the new room.

I wish we did more to preserve and continue your current room join across upgrades on way or another. That might mean any of:

* Auto-join the new room
* Shout more in the client (maybe highlight red) rooms with unfollowing upgrades

There are probably other possible approaches as well.

**Comment by aaronraimist** [comment]
Related: https://github.com/vector-im/element-web/issues/11932

99% sure they were red when room upgrades were first introduced but it broke

**Comment by Palid** [comment]
Related: https://github.com/vector-im/element-web/issues/19112

**Comment by Palid** [comment]
Maybe red highlight + moving those rooms to the top of the list, as an information that action is required would be __good enough__ for now?
